src/app/shared/group-title/group-title.component.pug

```diff
@@ -0,0 +1,11 @@
+.row.group-border
+    .col-9.pb-24.pt-24.z-1000
```

Comment:
@jialin-he I'm curious about the z index 1000 here.

Reply:
If I don't set a higher z-index here, the content will be covered by the pseudo element created by ":after" in the css. (And therefore you cannot select the words with your mouse.)